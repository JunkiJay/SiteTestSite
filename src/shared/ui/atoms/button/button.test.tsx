import { render, screen } from '@testing-library/react';
import { Button } from '.';

describe('Button component', () => {
  it('Renders correctly', () => {
    const { container } = render(<Button>Test</Button>);

    expect(screen.getByRole('button')).toHaveTextContent('Test');

    expect(container).toMatchSnapshot();
  });
});
