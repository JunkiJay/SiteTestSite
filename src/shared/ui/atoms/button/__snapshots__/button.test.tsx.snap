// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Renders correctly 1`] = `
<div>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary css-1o1c5h5-MuiButtonBase-root-MuiButton-root"
    tabindex="0"
    type="button"
  >
    Test
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </button>
</div>
`;
